stats/view_descriptor_constants.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016, Google Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.instrumentation;
+
+import "stats/census.proto";
+
+option java_package = "com.google.instrumentation.stats.proto";
+option java_outer_classname = "ViewDescriptorConstantsProto";
+
+// Constants for defining a specific set of view descriptors.
+// This data structure is used to convert pre-defined constants from/to
+// text-format proto files.
+// - To generate a binary proto file (*.pb) from a text-format proto file
+//   (*.textpb), use:
+//     BASE="filename"  (for example, BASE="rpc_constants")
+//     cat $BASE.textpb | \
+//     protoc --encode=google.instrumentation.ViewDescriptorConstants \
+//     view_descriptor_constants.proto > $BASE.pb
+// - To convert a binary proto file back to a text-format proto file, use:
+//     BASE="filename"  (for example, BASE="rpc_constants")
+//     cat $BASE.pb | \
+//     protoc --decode=google.instrumentation.ViewDescriptorConstants \
+//     view_descriptor_constants.proto > $BASE.textpb
+message ViewDescriptorConstants {
```

Comment:
Yes I suppose so, in the future we may want to  support pre-defined view constants for other services, and the ViewDescriptorConstants could be reused. Besides, if we don't have this data structure, the rpc_constants.textpb will need to be divided into a lot of separate files (in order to be parsed individually as separate MeasurementDescriptor and ViewDescriptor).